packages/map-template/src/components/Map/Map.jsx

```diff
@@ -248,5 +260,5 @@ function getVenueToShow(preferredVenueName, venues) {
     }
 
     // Else take first venue sorted alphabetically
-    return venues.sort(function (a, b) { return (a.name > b.name) ? 1 : ((b.name > a.name) ? -1 : 0); })[0];
+    return [...venues].sort(function (a, b) { return (a.name > b.name) ? 1 : ((b.name > a.name) ? -1 : 0); })[0];
```

Comment:
Why was this change necessary here?

Reply:
`sort` modifies the venues variable, which is now not allowed due to it being held in a Recoil atom.